### DISCUSSION TITLE
[4.2] [Workflow] Allow a transition to go from multiple Current Stages to a Target Stage

### DISCUSSION THREAD

**Discussion by bembelimen** [joomla/joomla-cms#47162] [3. Approved Features]
### Is your feature request related to a problem? Please describe.
At the moment only 1:1 or All:1 connection for transitions is possible. A n:1 would ne nice.

### Additional context
See https://github.com/bembelimen/joomla-cms/issues/86 for full description

# Approval Information
Approved: 18.03.2026
Minimum Target Version: 6.2

**Comment by HLeithner** [comment]
**The Joomla Maintainer Team decided at the 18.03.2026 meeting to accept this feature.**
